test/mocks/xhr.js

```diff
@@ -72,6 +72,7 @@ function mockFetchServer() {
         }
       }),
       withCredentials: request.credentials === 'include',
+      keepalive: request.keepalive,
```

Comment:
This came to my attention only now and I'm sorry that I'm suggesting this so late, but:

 - this should be available under `request.fetch.request.keepalive`;
 - I think it'd be better for your keepalive request to use the fetch API directly (`ajax.js` exports it as `fetch`); since it's not a "normal" bid request, it doesn't need the XHR shim (which is needed because the rest of core was written to use XHR), plus keepalive doesn't really make sense for "normal" bid requests so in my view it's a bonus if that remains unavailable to them.